src/views/Search.vue

```diff
@@ -7,6 +7,13 @@
     </ion-header>
     <ion-content>
       <ion-searchbar @ionFocus="selectSearchBarText($event)" v-model="queryString" :placeholder="$t('Search')" @keyup.enter="queryString = $event.target.value; searchProducts()"/>
+      <!-- Empty state -->
+      <div class="empty-state"  v-if="!products.length">
+        <!-- No result -->
+        <p v-if="products.length===0">{{ $t("No results found")}}</p>
```

Comment:
v-if="products.length===0" is not required here, it is already in div with the condition. And improve the logic of displaying the "No results found" message. It should be diplayed only when the user searches for a product not initially.